soh/soh/Enhancements/audio/AudioEditor.cpp

```diff
@@ -27,7 +27,7 @@ s8 reverbAdd = 0;
 #define SEQ_COUNT_BGM_EVENT 17
 #define SEQ_COUNT_INSTRUMENT 6
 #define SEQ_COUNT_SFX 57
-#define SEQ_COUNT_VOICE 107
+#define SEQ_COUNT_VOICE 108
```

Comment:
Hard to tell from the collection as its completely shuffled, but what new voice was added? The PR doesn't mention adding a new voice to the randomized pool.

Reply:
I assumed the new voice was the mentioned `Adult Link - Drinking`

Reply:
Ah the PR description called it a "sound effect" so I assumed sfx. In that case then wouldn't `SEQ_COUNT_SFX` need to increase too because of hovers?

But then again, both of those two added are considered `execludeAsReplacement` and probably should not be added to these count values.

Reply:
I'm honestly not sure why I only needed to up the Voice amount, but it was crashing if I didn't do that (but sound effects are fine). I think it needs to be increased because the voice effect can also pick itself.

I'll dig into this when I have time so I have a more concrete answer.

Reply:
So looks like before this PR, there was actually only 56 SFX, and that's why I didn't need to increase it.